cart/cart.css

```
@@ -0,0 +1,95 @@
+<link href="https://fonts.googleapis.com/css2?family=Tenor+Sans&display=swap" rel="stylesheet">
+
```

we add the `<link>` for the font inside the head tag in the HTML file, if you wanna add it to CSS you should use `@import` 

```suggestion
@import url('https://fonts.googleapis.com/css2?family=Tenor+Sans&display=swap');
```